lib/utils.rb

```diff
@@ -178,14 +179,20 @@ def in_dir(path, expand: true, silent: false)
     result
   end
 
-  def experiment_cofig(name)
+  def experiments
     @experiments ||= parse_json("config/experiments.json")
-    @experiments.find { |experiment| experiment["name"] === name }
+  end
+
+  def experiment_config(name)
+    experiments.find { |experiment| experiment["name"] === name }
   end
 
   def each_experiment
-    @experiments ||= parse_json("config/experiments.json")
-    @experiments.each { |experiment| yield experiment }
+    experiments.each { |experiment| yield experiment }
+  end
+
+  def custom_ember_glimmer
+    @custom_ember_glimmer ||= experiments.all? { |e| e["ember"] && e["glimer"] }
```

Comment:
Lets base this off of `e['ember']` only, inside the `build_ember` script I am going to make the glimmer build optional separately